robotiq_force_torque_sensor/src/rq_sensor_state.cpp

```diff
@@ -194,6 +194,36 @@ float rq_state_get_received_data(UINT_8 i)
 	}
 }
 
+
+
+/**
+ * \fn bool rq_state_get_command(INT_8 command, char *value)
+ * \brief Gets the value of high level information from the sensor
+ * \param command has to be in [1,2,4] corresponding to SerialNumber, FirmwareVersion and ProductionYear
+ * \param value A string
+ * \return true iff command is valid
+ */
+bool rq_state_get_command(INT_8 command, INT_8 * const  value)
+{
+	/// values correnspond to constants in sensor_accessor.srv
+	switch (command) {
+	case 1:
```

Comment:
How about using a local ENUM instead of magic numbers then.  Please follow the [ROS standard](http://wiki.ros.org/CppStyleGuide#Enumerations) on Enumerations.